numpy/ma/core.py

```diff
@@ -3017,7 +3020,9 @@ def __array_wrap__(self, obj, context=None):
                     except KeyError:
                         # Domain not recognized, use fill_value instead
                         fill_value = self.fill_value
-                    result = result.copy()
+                    if result is not self:
+                        # MHvK: Not sure why this copy is wanted.
+                        result = result.copy()
```

Comment:
I just spent a little bit of effort trying to figure out if this copy had any purpose, but I can't see any. It's not related to the copying of masks discussed in #5580, for example.

Want to just remove it?